src/en/help-openstack.md

```diff
@@ -20,6 +20,10 @@ juju add-cloud
 ```
 ...will enter the interactive mode. Enter the desired values to continue.
 
+!!! Note:
+To use the authentication from OpenStack, find the URL in your novarc the
```

Comment:
Not sure, I've not tried it.


Okay, looking at it again, I think this info should go just after the sentence "...regions to use." 

And probably say 

If you are unsure about endpoints and regions, they will be listed in the .novarc file for you OpenStack, if you have one.

Does that make it clearer. I mean, it doesn't even need to be a note, it could be in brackets at the end of that sentence